gui/itemStats.py

```diff
@@ -856,7 +856,7 @@ def OnRightClick(self, event):
         elif 'wxMac' in wx.PlatformInfo:
             os.system("open " + file_)
         else:
-            subprocess.call(["xdg-open", file_])
+            subprocess.call({"xdg-open": file_})
```

Comment:
I'm looking at the docs and they're telling me this is supposed to be a list, not a dict

https://docs.python.org/2/library/subprocess.html

Reason for this change?